backend/models/df.py

```diff
@@ -67,5 +66,5 @@ class ClusterUsageStatsModel(BaseModel):
     pools: List[PoolStatsModel]
 
     @staticmethod
-    def get(mgr: MgrModule) -> ClusterUsageStatsModel:
+    def get(mgr):  # type: (MgrModule) -> ClusterUsageStatsModel
```

Comment:
Any reason why you are using old-style notation here?

Reply:
Because we can't use Forward Reference annotations with python3.6, and we're returning a symbol that has not been defined yet.